src/seedu/addressbook/data/person/Address.java

```diff
@@ -38,19 +46,21 @@ public static boolean isValidAddress(String test) {
 
     @Override
     public String toString() {
-        return value;
+        String stringAddress = block.getBlock() + ", " + street.getStreet() + ", " + 
+                unit.getUnit() + ", " + postalCode.getPostalCode();
```

Comment:
Recall from the tutorial about the improvement that can be made to make this more readable.